SamplifyAPIClientTest/Tester.cs

```diff
@@ -4,110 +4,137 @@
 using Newtonsoft.Json;
 using Dynata.SamplifyAPIClient;
 using Xunit;
+using System.IO;
+
 namespace SamplifyAPIClientTest
 {
     public class TempTest
     {
         [Fact]
-        public async Task TestAll()
+        public async void TestFoo()
         {
-            Random random = new Random();
-            string projId = "projectTest" + random.Next();
             Trace.Listeners.Add(new TextWriterTraceListener(Console.Out));
             Trace.AutoFlush = true;
-            SamplifyClient testClient = new SamplifyClient("api", "samplifyweb", "samplifyweb", SamplifyEnv.UAT);
-
-            //// WORKS
```

Comment:
Can we remove all the comments/commented out code etc.? Also, should this file have the samplifyweb credentials exposed in it? In fact, I'm pretty sure this file shouldn't be here at all.

Reply:
Okay. I thought I removed that last Friday